changelogs/0.13.md

```diff
@@ -0,0 +1,87 @@
+# [0.13.0]
+
+## Added
```

Comment:
describe for what are needed all added or add links to another page with full description